### DISCUSSION TITLE
Is there a way to load the minified Keycloak JS adapter from Keycloak directly?

### DISCUSSION THREAD

**Discussion by moritzschmitz-oviva** [keycloak/keycloak#27812] [Q&A] [answered]
We have some older code where we always update the *.js file manually whenever there is a new Keycloak release. I was wondering whether Keycloak serves these files itself?

**Comment by jonkoops** [comment, marked as answer]
Yes, [see the documentation](https://www.keycloak.org/docs/latest/securing_apps/#installation-2). But we don't recommend loading Keycloak JS from the sever, instead you should use a package manager such as NPM.

**Reply by moritzschmitz-oviva** [reply]
Makes sense. Thanks!
